src/cdk/tree/nested-node.ts

```diff
@@ -49,6 +50,7 @@ import {getTreeControlFunctionsMissingError} from './tree-errors';
     '[attr.aria-expanded]': 'isExpanded',
     '[attr.role]': 'role',
     'class': 'cdk-tree-node cdk-nested-tree-node',
+    '(focus)': 'focus()',
```

Comment:
Doesn't this have the potential for an infinite loop? E.g. the `focus` event is fired which focuses the element which causes it to fire the `focus` event again.